src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
You have mentioned a fallback font which is good. But it's always better to include more than one fallback fonts. In case 'Roboto' font fails to load, the browser will use 'sans-serif'. But 'sans-serif' is a wide range of fonts, and the browser can pick any it likes. It's better to specify one more exact fallback font before 'sans-serif'. It's a minor issue, but it's better to follow best practices.